diff --git a/eeschema/sim/sim_plot_frame.cpp b/eeschema/sim/sim_plot_frame.cpp
index 70491a4..651474a 100644
--- a/eeschema/sim/sim_plot_frame.cpp
+++ b/eeschema/sim/sim_plot_frame.cpp
@@ -167,15 +167,15 @@ SIM_PLOT_FRAME::SIM_PLOT_FRAME( KIWAY* aKiway, wxWindow* aParent ) :
 
     // Toolbar buttons
     m_toolSimulate = m_toolBar->AddTool( ID_SIM_RUN, _( "Run/Stop Simulation" ),
-            KiBitmap( BITMAPS::sim_run ), _( "Run Simulation" ), wxITEM_NORMAL );
+            wxBitmapBundle( KiBitmap( BITMAPS::sim_run ) ), _( "Run Simulation" ), wxITEM_NORMAL );
     m_toolAddSignals = m_toolBar->AddTool( ID_SIM_ADD_SIGNALS, _( "Add Signals" ),
-            KiBitmap( BITMAPS::sim_add_signal ), _( "Add signals to plot" ), wxITEM_NORMAL );
+            wxBitmapBundle( KiBitmap( BITMAPS::sim_add_signal ) ), _( "Add signals to plot" ), wxITEM_NORMAL );
     m_toolProbe = m_toolBar->AddTool( ID_SIM_PROBE,  _( "Probe" ),
-            KiBitmap( BITMAPS::sim_probe ), _( "Probe signals on the schematic" ), wxITEM_NORMAL );
+            wxBitmapBundle( KiBitmap( BITMAPS::sim_probe ) ), _( "Probe signals on the schematic" ), wxITEM_NORMAL );
     m_toolTune = m_toolBar->AddTool( ID_SIM_TUNE, _( "Tune" ),
-            KiBitmap( BITMAPS::sim_tune ), _( "Tune component values" ), wxITEM_NORMAL );
+            wxBitmapBundle( KiBitmap( BITMAPS::sim_tune ) ),  _( "Tune component values" ), wxITEM_NORMAL );
     m_toolSettings = m_toolBar->AddTool( wxID_ANY, _( "Sim Command" ),
-            KiBitmap( BITMAPS::config ), _( "Simulation command and settings" ), wxITEM_NORMAL );
+            wxBitmapBundle( KiBitmap( BITMAPS::config ) ), _( "Simulation parameters and settings" ), wxITEM_NORMAL );
 
     // Start all toolbar buttons except settings as disabled
     m_toolSimulate->Enable( false );

